GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/LooseObjectStepTests.cs

```diff
@@ -62,13 +53,13 @@ public void PutLooseObjectsInPackFiles()
 
             // Expand one pack, and verify we have loose objects
             this.ExpandOneTempPack(copyPackBackToPackDirectory: false);
-            int loooseObjectCount = this.GetLooseObjectFiles().Count();
-            this.GetLooseObjectFiles().Count.ShouldBeAtLeast(1);
+            int looseObjectCount = this.GetLooseObjectFiles().Count();
+            looseObjectCount.ShouldBeAtLeast(1);
 
             // This step should put the loose objects into a packfile
             this.Enlistment.LooseObjectStep();
 
-            this.GetLooseObjectFiles().Count.ShouldEqual(loooseObjectCount);
+            this.GetLooseObjectFiles().Count.ShouldEqual(looseObjectCount);
             this.CountPackFiles().ShouldEqual(1);

```

Comment:
Should we verify that previuosly there were 0 packfiles?

Reply:
We do, it's just hidden in the diff collapse.